examples/create_identity_provider/htpasswd/README.md

```diff
@@ -6,7 +6,7 @@ Using htpasswd authentication in OpenShift Container Platform allows you to iden
 
 ## Prerequisites
 
-1. You created your [account roles using Terraform](../../examples/create_rosa_cluster/create_rosa_sts_cluster/classic_sts/account_roles/README.md).
```

Comment:
Updated this as i needed the link and it was incorrect :sweat_smile:

Reply:
Thank you for that.
The examples here are going to be replaced soon with the examples in the module.
https://github.com/terraform-redhat/terraform-rhcs-rosa-classic 
Please check this from the module as well